TeslaApp/TeslaApp/SettingsScreen/SettingsView.swift

```diff
@@ -0,0 +1,139 @@
+// SettingsView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Settings view
+struct SettingsView: View {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let teslaText = "Tesla"
+        static let mileageText = "187 km"
+        static let teslaImageName = "teslaMain"
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        backgroundStackView {
+            VStack {
+                headerView
+                carView
+                controlPanelView
+                Spacer()
+                    .frame(height: 40)
+                if settingsViewModel.tagSelected == 1 {}
+                goToAirSystemView
+                Spacer()
+            }
+        }
+        .navigationBarHidden(true)
+    }
+
+    // MARK: - Private Properties
+
+    @StateObject private var settingsViewModel = SettingsViewModel()
+
+    @State private var isAirSystemViewShown = false
```

Comment:
```suggestion
    @State private var isAirSystemViewShown = false
```

можно попробовать во вью модель вынести